src/resolvers/powerBoostingResolver.ts

```diff
@@ -132,13 +141,21 @@ export class PowerBoostingResolver {
     @Arg('percentage', type => Float) percentage: number,
     @Ctx() { req: { user } }: MyContext,
   ): Promise<PowerBoosting[]> {
-    if (!user || !user?.userId) {
+    const userId = user?.userId;
+    if (!user || !userId) {
       throw new Error(
         i18n.__(translationErrorMessagesKeys.AUTHENTICATION_REQUIRED),
       );
     }
 
-    return setSingleBoosting({ userId: user.userId, projectId, percentage });
+    const result = await setSingleBoosting({
+      userId,
+      projectId,
+      percentage,
+    });
+    const project = (await findProjectById(projectId)) as Project;
+    await getNotificationAdapter().projectBoosted({ project, userId });
```

Comment:
I guess it's not very cool to fetch project information here, we can rely on projectBoosted method to do it.
Also, and importantly, I think it's not cool to block API for sending a notification. We can do it in parallel, but if any error happens we must be sure will see it (collecting by sentry or anything else).
Technically, remove that `await` and handle it in `.catch` for example.

Reply:
Done, I didn't know what to do to not get a warning from IDE without disabling eslint, nice idea